pkg/urbit/vere/io/ames.c

```diff
@@ -472,29 +472,40 @@ _ames_sift_body(u3_head* hed_u,
                 c3_w     len_w,
                 c3_y*    bod_y)
 {
-  c3_y rog_y = ( c3y == hed_u->rel_o )? 6 : 0;
-  c3_y* gob_y = bod_y + rog_y;
-  c3_s gob_s = len_w - rog_y;
-
-  c3_y pre_y = _ames_sift_prelude(hed_u, &bod_u->pre_u, len_w, bod_y);
-
-  if (pre_y >= len_w ) {
-    return c3n;
-  }
-  else {
-    bod_u->mug_l = u3r_mug_bytes(gob_y, gob_s) & 0xfffff;
+  c3_w mod_w, hod_w;
+  bod_u->mug_l = u3r_mug_bytes(bod_y, len_w) & 0xfffff;
+  bod_u->con_y = c3_calloc(len_w);
+  memcpy(bod_u->con_y, bod_y, len_w);
+  mod_w = bod_u->mug_l & 0xfffff;
+  hod_w = hed_u->mug_l & 0xfffff;
+  return ( mod_w == hod_w ) ? c3y : c3n;
+}
 
-    bod_u->con_y = gob_y + 4;
+static void
+_ames_etch_short(c3_y buf_y[2], c3_s sot_s)
+{
+  buf_y[0] = sot_s & 0xff;
+  buf_y[1] = (sot_s >>  8) & 0xff;
+}
 
-    return c3y;
-  }
+static void
+_ames_etch_word(c3_y buf_y[4], c3_w wod_w)
+{
+  buf_y[0] = wod_w & 0xff;
+  buf_y[1] = (wod_w >>  8) & 0xff;
```

Comment:
I thought the sift implementations looked quite good, with the alignment.

```suggestion
  buf_y[0] = (wod_w >>  0) & 0xff;
  buf_y[1] = (wod_w >>  8) & 0xff;
```

```suggestion
  buf_y[0] =  wod_w        & 0xff;
  buf_y[1] = (wod_w >>  8) & 0xff;
```

Reply:
yes, will realign